docs/reference/api/docker_remote_api_v1.23.md

```diff
@@ -372,10 +372,14 @@ Json Parameters:
 -   **StopSignal** - Signal to stop a container as a string or unsigned integer. `SIGTERM` by default.
 -   **HostConfig**
     -   **Binds** – A list of volume bindings for this container. Each volume binding is a string in one of these forms:
-           + `host_path:container_path` to bind-mount a host path into the container
-           + `host_path:container_path:ro` to make the bind-mount read-only inside the container.
-           + `volume_name:container_path` to bind-mount a volume managed by a volume plugin into the container.
-           + `volume_name:container_path:ro` to make the bind mount read-only inside the container.
+           + `host_path:container_path[:options]` to bind-mount a host path into the container
+           + `volume_name:container_path[:options]` to bind-mount a volume managed by a volume plugin into the container.
+
+        `options` can be a comma-delimited list of:
+           + `[ro|rw]` to mount a volume read-only or read-write, respectively. By default, the volumes are mounted read-write.
+           + `[z|Z]` to indicate that two containers share the same volume or that a volume is private, respectively.
```

Comment:
This probably should mention "security labels"; some info can be found here https://github.com/docker/docker/blob/master/docs/reference/commandline/run.md#mount-volumes-from-container---volumes-from